packages/react-resizable-panels/src/utils/callPanelCallbacks.ts

```diff
@@ -32,6 +32,7 @@ export function callPanelCallbacks(
     const lastNotifiedMixedSizes = panelIdToLastNotifiedMixedSizesMap[panelId];
     if (
       lastNotifiedMixedSizes == null ||
+      lastNotifiedMixedSizes == undefined ||
```

Comment:
`== null` will match both `null` and `undefined` so this change is redundant.

Reply:
TIL, ty